plugins/flytekit-pydantic/setup.py

```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=1.7.0b0,<2.0.0", "pydantic"]
+plugin_requires = ["flytekit>=1.7.0,<2.0.0", "pydantic<2.0.0"]
```

Comment:
why do we need to change here

Reply:
Thank you for questioning this, just come out a better solution!